### PR TITLE
Jonathan Johnson - #HW1

### PR DESCRIPTION
**Pull request by j2jonathan952**
Jonathan Johnson - Comments and Reflection

In terms of content, I added a biography about my academic life, a short list of my general non-academia proclivities, and a fun section on joking-but-serious goals in my life, with GIFs underneath. I ended with a link to my LinkedIn if people wanted to learn more about me. 

Incorporating these elements was intuitive and easy given the guide provided (https://guides.github.com/features/mastering-markdown/#syntax). For almost everything, I simply followed this guide. I wanted to add GIFs (because it is more fun), so I found this website, which was helpful: https://blogdown-demo.rbind.io/2018/01/31/gif-animations/

I initially made a couple small errors, for example, not leaving a space after the #, which is how the program recognizes the text is a header. But, the preview button helped me catch the errors before submitting. However, I’m happy I checked the upload to mygithub because one of my image links did not load properly, even though it showed up in the preview. I fixed it in my local repository, restaged, and committed, then pushed it up to mygithub.

I only had one real struggle: trying to add an image NOT from online (i.e. R studio file or local desktop). I googled, looked at StackOverflow, tried to see if the knitr package would solve it, etc. I couldn’t set up the pathway properly, so I caved and uploaded the image to Imgur. One of the tutorials I read recommended imgur, and it was great because it came with all the relevant codes/links for R markdown already.  

The workflow was straightforward after watching the tutorial video (thank you!). I ended up committing a few too many times, but, otherwise, not a problem.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,33 @@
...
+2. Am great at eating any and all food, but terrible at cooking any and all food
+3. Enjoy spending time with friends and family and meeting new people
+
+### Passions and Goals in Life
+
+##### My true goal in life is simply to be Adele. She's a queen. 
+
+![](https://media.giphy.com/media/LlX5BmWRnEaUU/giphy.gif) 
+
+##### To grow another foot to become a professional volleyball player
```
Comment (ahyeoncho) [inline README.md:23]: Lol that's funny

File: README.md [hunk 2]
```diff
@@ -1 +1,33 @@
...
+
+1. Love to sing all the time, even though I'm not very good
+2. Am great at eating any and all food, but terrible at cooking any and all food
+3. Enjoy spending time with friends and family and meeting new people
+
+### Passions and Goals in Life
+
+##### My true goal in life is simply to be Adele. She's a queen. 
+
+![](https://media.giphy.com/media/LlX5BmWRnEaUU/giphy.gif)
```
Comment (ahyeoncho) [inline README.md:21]: I like your use of images.

File: README.md [hunk 3]
```diff
@@ -1 +1,33 @@
...
+
+![](https://i.imgur.com/5JGh0bLm.png)
+
+I am an anthropologist by training, but I enjoy learning new methods and techniques. At the University of Chicago, I focus primarily on child labor in Cambodia and youth's rights. 
+
+Outside of academia, I also:
+
+1. Love to sing all the time, even though I'm not very good
+2. Am great at eating any and all food, but terrible at cooking any and all food
+3. Enjoy spending time with friends and family and meeting new people
```
Comment (ahyeoncho) [inline README.md:15]: Use of list makes you seem very organized. Love it!

**Review by ahyeoncho — COMMENTED** [review]
Great job. Couldn't find anything negative to say :)

**Review by kaymoorewms — COMMENTED** [review]
Wow! This was incredibly done. Some highlights: 
1. Your comment was very thorough, and I am excited to use the gif feature from the website you linked!!!
2. This was easy to follow, well integrated, and you used all the elements very well
3. You clearly used trial and error and seemed to learn a lot from this assignment

I guess an area of improvement could be to restructure? Maybe put your Linkedin earlier on? But that is really all I can say because it is very well done!

File: README.md (hunk 2 above)
Comment (djanvier) [inline README.md:21]: Nice GIFS!

File: README.md [hunk 4]
```diff
@@ -1 +1,33 @@
-# hw01
\ No newline at end of file
+# Homework 1
+
+### About Me
+
+My name is **Jonathan Johnson** or *JJ* for short. 
+
+![](https://i.imgur.com/5JGh0bLm.png)
```
Comment (djanvier) [inline README.md:7]: This is a great alternative way to embed an image in Markdown. In order to embed in image locally, you can: 
1. Upload the image to your homework repository, or
2. Add the image to the same file as your README.md and push it to GitHub.

**Review by djanvier — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |    x       |              |            |
| **Reproducibility**         |     x      |              |            |
* Great job on this assignment! Good use of outside sources and great commit history

**Pull request closed without merging**